docker/README.md

```diff
@@ -60,7 +61,18 @@ REPOSITORY              TAG     IMAGE ID       CREATED          SIZE
 local/zkevm-contracts   local   54d894c6a5bd   10 minutes ago   2.3GB
 ```
 
-Here's a quick reference matrix for mapping fork IDs to branches/releases:
```

Comment:
is this an automatic process, i.e. when new tag created here: 0xPolygonHermez/zkevm-contracts we autogen one on our side ?

Reply:
It would be great to automate that honestly.